### PR TITLE
GitHub Actions: Be precise about the format

### LINKED ISSUE #476: Github Actions context variables data format not precise
### What article on docs.github.com is affected?

<https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions>

### What part(s) of the article would you like to see updated?

Be more precise about the data format on `github.ref` like on <https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables>

I thought I could use it as a tag for a docker build but this does not work. 

### Additional information

[PR](https://github.com/github/docs/pull/400) already opened

### PR DESCRIPTION
**Pull request by tillepille**
I stumbled across this to add the plain branch name to the [Docker push action](https://github.com/docker/build-push-action) as a tag, which does not work with 
```yaml
with:
  tags: tillepille/my-image: ${{ github.ref }}
```

### Why:
I tried to get the plain branch name as a variable in my Action. The documentation suggested to me that this is the plain branch name e.g. `master` but it is in the format of `refs/heads/master` which isn't applicable as e.g a docker tag.

### What's being changed:

Be more precise about the format of the variable.
I added in the form of `refs/heads/<BRANCH_NAME>` to the explanation.

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

closes #476

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a PR 👍 Please fill out the [pull request template](https://raw.githubusercontent.com/github/docs/main/.github/PULL_REQUEST_TEMPLATE.md) in the top comment in this pull request, then we'll get this up for review!

**Comment by tillepille** [comment]
@janiceilene Sorry, I missed it when opening the mr via whole UI workflow. Filled out, Issue also created

**Comment by janiceilene** [comment]
Thanks so much for filling out the template and opening an issue @tillepille 💛 I'll get this triaged for review!

File: content/actions/reference/context-and-expression-syntax-for-github-actions.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ The `github` context contains information about the workflow run and the event t
 | `github.event_path` | `string` | The path to the full event webhook payload on the runner. |
 | `github.head_ref` | `string` | The `head_ref` or source branch of the pull request in a workflow run. This property is only available when the event that triggers a workflow run is a `pull_request`. |
 | `github.job` | `string` | The [`job_id`](/actions/reference/workflow-syntax-for-github-actions#jobsjob_id) of the current job. |
-| `github.ref` | `string` | The branch or tag ref that triggered the workflow run. |
+| `github.ref` | `string` | The branch or tag ref that triggered the workflow run in the form of `refs/heads/<BRANCH_NAME>`. |
```
Comment (lucascosti) [inline content/actions/reference/context-and-expression-syntax-for-github-actions.md:96]: Thanks for this PR, @tillepille! Your change covers the format for branches, but I'll also add the format for tags too 🙂 

```suggestion
| `github.ref` | `string` | The branch or tag ref that triggered the workflow run. For branches this in the format  `refs/heads/<branch_name>`, and for tags it is `refs/tags/<tag_name>`. |
```

**Review by lucascosti — APPROVED** [review]
🚢

File: content/actions/reference/context-and-expression-syntax-for-github-actions.md (hunk 1 above)
Comment (tillepille) [reply content/actions/reference/context-and-expression-syntax-for-github-actions.md]: I didn't test that. Thank you @lucascosti  !

Comment (lucascosti) [reply content/actions/reference/context-and-expression-syntax-for-github-actions.md]: No worries, thanks a lot for the contribution! 👍

**Pull request merged**